### DISCUSSION TITLE
İbax perfect

### DISCUSSION THREAD

**Discussion by Skips1** [IBAX-io/go-ibax#2080] [Ideas] [answered]
Karanlıklardan doğar aydınlıklar???
ddd

> [``](![Gülümse](https://user-images.githubusercontent.com/98268472/206925221-c3cc429b-7669-48aa-810f-7a044f269ca8.jpg))

**Comment by Skips1** [comment]
Hmm

**Comment by enzifiri** [comment, marked as answer]
Gece-geceler
Geceler
Geceler

**Comment by MrDino3813** [comment]
Ohhh really I'll check them out and give you a reply
